robot_skills/src/robot_skills/mockbot.py

```diff
@@ -45,22 +46,81 @@ def old_query(spec, choices, timeout=10):
     raise Exception('robot.ears.recognize IS REMOVED. Use `robot.hmi.query`')
 
 
+class AlteredMagicMock(mock.MagicMock):
```

Comment:
I created an Altered magicMock class to allow us to create less strict tests. This way a call:
mock.('argument 1', 'argument_2', kwarg1='keywordargument 1') will satisfy
mock.assert_called_with('argument 1'). This way we can relax requirements on the amount of arguments passed. So you can rely on the default arguments or you can add your own. This mock allows both

Reply:
Maybe we could monkey patch MagicMock

Reply:
I don't think monkeypatching would be elegant in this scenario